### ISSUE TITLE
[Theme Handbook] WordPress 6.4 - 6.6 updates

### ISSUE THREAD

**Issue by carolinan** [WordPress/Documentation-Issue-Tracker#1414]
Let's have an early look at what updates may be needed when 6.5 is released.
**This is of course tentative until we are closer to release,** and not exhaustive. Feel free to edit this issue and add more items, but also suggestions of which document this should be added to.

- [ ] Appearance tools theme support (Classic). Decide if this should be mentioned, or if the developer documentation for the add_theme_support() is enough.
- [ ] The order the CSS is loaded in is updated, follow: https://core.trac.wordpress.org/ticket/60280
- [ ] Theme.json: 
  - [ ]  All theme.json version 3 changes in 6.6.
   - [ ] Add defaultSpacingSizes option (theme.json v3) https://github.com/WordPress/gutenberg/pull/61842/
  - [ ] allowCustomContentAndWideSize: https://github.com/WordPress/gutenberg/pull/56236
  - [ ] Settings: Heading and button colors https://github.com/WordPress/gutenberg/pull/55674
  - [ ] backgroundImage https://github.com/WordPress/gutenberg/pull/53934
  - [ ] dimensions.aspectRatio: https://github.com/WordPress/gutenberg/pull/56897

- <del>disable default font size: https://github.com/WordPress/gutenberg/pull/56661 https://github.com/WordPress/gutenberg/pull/58456  
  </del>
- <del>FontFamilies preview: https://github.com/WordPress/gutenberg/pull/56793</del>

**Comment by carolinan** [comment]
@justintadlock  There also seems to be some 6.4 features missing? I was looking for information about block hooks.

**Comment by justintadlock** [comment]
Yeah, we never did a ticket like this for 6.4. So, we might want to make this "WordPress 6.4 + 6.5 updates". :) 

I know the `backgroundImage` setting needs to be added. I'll do a review in the next few days to see if there are other things.

**Comment by justintadlock** [comment]
Added aspectRatio to the list.

**Comment by carolinan** [comment]
theme.json FontFamilies preview was removed

**Comment by justintadlock** [comment]
Some more that popped up in Gutenberg 17.7 RC1 that I haven't had time to investigate:

- https://github.com/WordPress/gutenberg/pull/57982
- https://github.com/WordPress/gutenberg/pull/58142
- https://github.com/WordPress/gutenberg/pull/58592
